view collectible

### Description

https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=6455%3A122769

Implements the ERC721 view as in design

Acceptance criteria:

- Header shows:
-- Collection icon
-- Collection name
-- Collectible ID
-- Network tag (Icon + name)
- Body shows:
-- Collectible name
-- Collectible description
- Property names in original case (as delivered by OpenSea)

There should be a "compact" mode when the window is narrow, and a "normal" mode when the windows is wide, altering:
- General margins
- Header items positioning+sizes
- Collecctible image size

Current:

![Screenshot_20221115_111308](https://user-images.githubusercontent.com/11161531/201942923-8e28525c-dc5e-4b3a-aa20-c6cbfb522d1c.png)

Model:

![Screenshot_20221115_111539](https://user-images.githubusercontent.com/11161531/201943004-d402014f-a221-40d7-ac78-e7601d58cf7b.png)
![Screenshot_20221115_111559](https://user-images.githubusercontent.com/11161531/201943019-eaafc97c-f7ca-496c-954c-4bb5dba9d83b.png)
![Screenshot_20221115_111615](https://user-images.githubusercontent.com/11161531/201943039-f53243d2-35ea-46b7-af0a-6781bab60b1c.png)

Visual changes:

- General
  - Header
  1. Replace Collection icon with Status icon?
  2. Replace Collectible name with Collection name
  3. Add network icon + name
  - Body
  1. Add title with Collectible name
  2. In Narrow mode, smaller ID, next to network icon
  3. In Narrow mode, smaller Collectible image
  - Properties
  1. Property names not in uppercase
  - Activity
  1. Add Activity tab

Bugs:

- Fix crash

Steps to reproduce:
1. Setup wallet with two view-only accounts with collectibles (examples: `0x5aB130251D128bc10Df5F77573A8983307822d36`, `0x648550Eaca1654A1e65B13cf385Ef9Ff505dE909`, `0xb9eB79E3E735Ee636255dD8D65872a1287744e33`)
4. Go to the `Collectibles` tab of one account, expand one of the collections, click one of the collectibles.
5. Click other accounts.
After switching accounts a couple of times, the application will crash
```
DBG 2022-11-15 11:06:54.318-03:00 NewBE_callPrivateRPC                       topics="rpc" tid=204082 file=core.nim:20 rpc_method=wallet_getOpenseaAssetsByOwnerAndCollection
qml: Fetching the-bright-benzi-bananas-ai-pass
DBG 2022-11-15 11:06:55.938-03:00 NewBE_callPrivateRPC                       topics="rpc" tid=204082 file=core.nim:20 rpc_method=wallet_getOpenseaAssetsByOwnerAndCollection
qrc:/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml:47:17: Unable to assign [undefined] to double
qrc:/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml:47:17: Unable to assign [undefined] to double
qrc:/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml:47:17: Unable to assign [undefined] to double
qrc:/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml:47:17: Unable to assign [undefined] to double
qrc:/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml:47:17: Unable to assign [undefined] to double
qrc:/imports/shared/controls/InformationTag.qml:34:9: QML Image: Cannot open: qrc:/imports/assets/icons/tiny/.svg
DBG 2022-11-15 11:06:59.272-03:00 NewBE_callPrivateRPC                       topics="rpc" tid=204109 file=core.nim:20 rpc_method=wallet_getOpenseaCollectionsByOwner
qrc:/imports/shared/controls/InformationTag.qml:34:9: QML Image: Cannot open: qrc:/imports/assets/icons/tiny/.svg
qrc:/imports/shared/controls/InformationTag.qml:34:9: QML Image: Cannot open: qrc:/imports/assets/icons/tiny/.svg
qrc:/imports/shared/controls/InformationTag.qml:34:9: QML Image: Cannot open: qrc:/imports/assets/icons/tiny/.svg
Traceback (most recent call last)
/home/dlipicar/repos/status-desktop/src/nim_status_client.nim(181) nim_status_client
/home/dlipicar/repos/status-desktop/src/nim_status_client.nim(178) mainProc
/home/dlipicar/repos/status-desktop/vendor/nimqml/src/nimqml/private/qguiapplication.nim(29) exec
/home/dlipicar/repos/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(49) qobjectCallback
/home/dlipicar/repos/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(34) onSlotCalled
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [Makefile:631: run-linux] Error 1
```

--- a/services/wallet/opensea.go
+++ b/services/wallet/opensea.go
@@ -169,7 +169,7 @@ func (o *OpenseaClient) fetchAllAssetsByOwnerAndCollection(owner common.Address,
 
 		for _, asset := range container.Assets {
 			for i := range asset.Traits {
-				asset.Traits[i].TraitType = strings.Replace(strings.ToUpper(asset.Traits[i].TraitType), "_", " ", 1)
+				asset.Traits[i].TraitType = strings.Replace(asset.Traits[i].TraitType, "_", " ", 1)
 				asset.Traits[i].Value = TraitValue(strings.Title(string(asset.Traits[i].Value)))
 			}
 			assets = append(assets, asset)

